.github/workflows/build-and-test.yml

```
@@ -22,7 +22,7 @@ jobs:
         shell: bash -l {0}
     strategy:
       matrix:
-        target_arch: ["linux-64", "linux-aarch64", "linux-ppc64le"]
+        target_arch: ["linux-64", "linux-aarch64", "linux-ppc64le", "osx-arm64", "osx-64"]
```

Looks like I'll have to take a look at arm64 but we can add just intel macs for now:
```suggestion
        target_arch: ["linux-64", "linux-aarch64", "linux-ppc64le", "osx-64"]
```